client/src/pages/Results.js

```diff
@@ -96,9 +96,9 @@ export function Results() {
                   </p>
                   <p>
                     <img src={DateSrc} alt="calendar icon" />
-                    {result.pickup_date}&nbsp;
+                    {result.date}&nbsp;
```

Comment:
Similar to dennis comment.
Whitespaces or linebreaks are not used for styling. 
Instead, use margin or padding.
You can wrap your date in an `span` element and style it.

Reply:
True. Fixed it in my last commit. Thank you!